src/index.html

```diff
@@ -5,8 +5,352 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <a href="index.html" class="header__logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+
+      <nav class="header__nav">
+
+        <ul class="header__links">
+
+          <li class="header__li">
```

Comment:
```suggestion
    <header class="header">
      <a href="index.html" class="header__logo">
        <img src="images/logo.png" alt="logo">
      </a>

      <nav class="header__nav">
        <ul class="header__links">
          <li class="header__li">
```
don't add spaces between parent and child elements, add them between blocks of code

Reply:
2'nd point in checklist